src/main/java/team4/footwithme/chat/api/ChatApi.java

```diff
@@ -1,11 +1,45 @@
 package team4.footwithme.chat.api;
 
+import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Slice;
+import org.springframework.data.domain.Sort;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.web.bind.annotation.*;
+import team4.footwithme.chat.api.request.ChatRequest;
+import team4.footwithme.chat.service.ChatService;
+import team4.footwithme.chat.service.response.ChatResponse;
+import team4.footwithme.global.api.ApiResponse;
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping
+@RequestMapping("/api/v1")
 public class ChatApi {
+    private final ChatService chatService;
+
+    /**
+     * 채팅 보내기
+     */
+    // TODO : test를 위해 email을 변수로 설정함. 나중에 매개변수에 @AuthenticationPrincipal에서 이메일 빼오기
+    @MessageMapping("/chat/message")
+    public void sendMessage(@Valid ChatRequest request) {
+        String email = "a@a.com";
+        chatService.sendMessage(request.toServiceRequest(), email);
+    }
+
+    /**
+     * 채팅 조회
+     * @param chatroomId 채팅방 ID
+     * @param page 현재 페이지
+     * @param size 한페이지에 나타날 갯수
+     * @return
+     */
+    @GetMapping("/chat/message/{chatroomId}")
+    public ApiResponse<Slice<ChatResponse>> getChatting(@PathVariable Long chatroomId, @RequestParam int page, @RequestParam int size) {
```

Comment:
request param에 필수 여부 추가해봐도 좋을 것 같아요 !
```suggestion
    public ApiResponse<Slice<ChatResponse>> getChatting(@PathVariable Long chatroomId, @RequestParam(required = false) int page, @RequestParam(required = false) int size) {
```

Reply:
좋은 생각입니다 감사합니다!